content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Upcoming ROS 1 End of Life"
+author:  Kat Scott 
+subtitle: Migration Paths for ROS 1 Users
+created_at: 2024-12-16 16:21:05 -0700
+kind: article
+header_class:  bg-photo 
+header_image:  /imgs/eol-bg.png 
+---
+
+## ROS 1 Noetic Ninjemys goes End-of-Life 2025-05-31
+
+When we [announced](https://www.openrobotics.org/blog/2020/5/23/noetic-ninjemys-the-last-official-ros-1-release) ROS 1 Noetic Ninjemys on May 23rd, 2020 we made it very clear that it would be the last official ROS 1 distribution. 
+In the proceeding five years we focused our efforts on making ROS 2 a world class framework for professional robot development, and our user community seems to agree.
+According to September 2024 download data from our servers ROS 2 downloads now make up almost 80% of all ROS downloads.
+We've now seen ROS 2 used successfully in production by a large number of companies, across dozens of industries, and we think it is time for the last remaining ROS users to make the switch. 
+
+If you have been directed to this page it was probably because you saw one of the notifications we slipped into the source code for ROS Noetic or RViz. 
+We've heard from a number of users that they were unaware that ROS 1 Noetic, and consequently all of 1, will go end-of-life on **May 31st, 2025**.
+We're sorry that we interrupted your work, but it is crucial that you know what happens once ROS 1 goes end-of-life.
```

Comment:
I think we could omit this paragraph (but keep the may 31st 2025 date somewhere)